src/SFA.DAS.Reservations.Web/Views/ProviderReservations/ProviderFundingPaused.cshtml

```diff
@@ -0,0 +1,18 @@
+﻿
+@{
+    ViewData["Title"] = "ProviderFundingPaused";
+}
+
+<div class="govuk-width-container">
+    <a href="#" class="govuk-back-link">Back</a>
```

Comment:
Will need to make this also go back to the same place as the link below:

Also, this needs to sit above the `<main>` tag